RandLAPACK/comps/rl_util.hh

```diff
@@ -82,21 +82,19 @@ void disp_diag(
     }
 }
 
-/// Extracts the l-portion of the GETRF result, places 1's on the main diagonal.
-/// Overwrites the passed-in matrix.
-template <typename T> void get_L( int64_t m, int64_t n, std::vector<T>& L);
-
 template <typename T>
 void get_L(
     int64_t m,
     int64_t n,
-    std::vector<T>& L
+    std::vector<T>& L,
+    int overwrite_diagonal
```

Comment:
C++ supports default arguments. So you can just set this to ``int overwrite_diagonal = true`` or ``int overwrite_diagonal = false`` depending on which you think is more common.

Reply:
You had a nice comment on lines 85 and 86. Why'd you remove those?

Reply:
I don't think i see a reason behind using a default here.